### ISSUE TITLE
Dots in Filenames Cause Errors

### ISSUE THREAD

**Issue by flolu** [thesayyn/protoc-gen-ts#17]
It seems as if dots in filenames cuase errors:

``` 
load("@rules_proto//proto:defs.bzl", "proto_library")
load("@npm//protoc-gen-ts:index.bzl", "ts_proto_library")

package(default_visibility = ["//visibility:public"])

# works fine
proto_library(
    name = "message_proto",
    srcs = ["message.proto"],
)

ts_proto_library(
    name = "message",
    deps = [
        ":message_proto",
    ],
)

# doesn't work
proto_library(
    name = "other_message_proto",
    srcs = ["other.message.proto"],
)

ts_proto_library(
    name = "other_message",
    deps = [
        ":other_message_proto",
    ],
)
```

The second one throws:
```
ERROR: /home/flolu/Desktop/protoc-gen-ts-filename-repro/BUILD.bazel:23:17: output 'other.message.ts' was not created
ERROR: /home/flolu/Desktop/protoc-gen-ts-filename-repro/BUILD.bazel:23:17: not all outputs were created or valid
```

### I've made a small repro here: https://github.com/flolu/protoc-gen-ts-filename-repro

**Comment by thesayyn** [comment]
Thank you for the reproduction. I'll take look into it probably this weekend.

**Comment by thesayyn** [comment]
Released in 0.3.2
